### ISSUE TITLE
detail and form view into card ?

### ISSUE THREAD

**Issue by scramatte** [Muetze42/nova-resource-card#5]
Hi 
Does it possible to show detail or form view into a card too ?

Regards

**Comment by Muetze42** [comment]
Not yet. So the detail view would definitely be conceivable. 

A shape is already more difficult. Should it be an edit view?

**Comment by scramatte** [comment]
Hi,

Yes, I mean edit and detail view.
Overall what I would like to be able to do is tune the detail vue to be able to adds cards anywhere (top, bottom, left or right side). Currently cards can be added only on top

**Comment by basserd** [comment]
Hi @Muetze42 I would also really be interested in a possibility of moving the card to the bottom of a resource. In my case I'm having a resource called "Material Batch" this batch is connected to other resources through a pivot table ("Basic recipe", "Color recipe" & "Activation recipe"). These three resources can be selected within a "Final recipe" resource. 

What I would like to achieve is to show in what "Final recipes" the "Material Batch" is indirectly selected. I've searched for a lot of ways to do this with the Laravel Nova relationship field functions, but it seems like it's not possible.

Therefore I have high hopes for your package, if I would be able to move the card to the end of the resource detail page that would be huge.

**Comment by Muetze42** [comment]
@basserd This would be a extra package to moves card to the bottom of a resource. I am not aware of any that can do this. Maybe have a look at NovaPackages?

**Comment by basserd** [comment]
@Muetze42 Fair enough! I have not found any that does anything like this. Only solution for now that I have is moving the card with Javascript haha.. which isn't ideal ofcourse.

**Comment by Muetze42** [comment]
Such ideas never leave me alone and stay in the back of my mind 😅

The file `resources/js/views/Detail.vue` contains:

```vue
    <div v-if="shouldShowCards && hasDetailOnlyCards">
      <Cards

    <!-- ... -->
      <component
```

For example, you could now expand the card and add a "bottom" option and install the whole thing under the component.

I've just always been a bit more careful with customizing Nova's own Vue components, as there are always a lot of changes with the updates.

I even have a special tool in the pipeline which should always inform you of a Nova update and show you the changes straight away... As I always have to worry about changes myself with Nova Menu and the like.

**Comment by basserd** [comment]
Thanks for informing me of that solution! Although it is risky to edit the Vue components as you said, I still think it's better than using Javascript. I'll just add a note that whenever updating composer I check whether this part still works haha.

Thanks alot! Have a good day :)

**Comment by Muetze42** [comment]
Actually, I'm currently working on an error tracker where only the notifications are missing. But this will not be public or commercial anyway...

I'm going to make the "Nova Watcher" now. This will be a public repo so that everyone can work with it and then it will be easier for all Nova developers to be informed about changes.

**Comment by Muetze42** [comment]
Forgot to mention here.

I have now developed a small tool. Features:

- Compare Nova Versions
- Notifications when there is a Nova update (can be activated when you are logged in)
- Compare files and see changes (if you have validated your Nova licence)

[NovaWat.ch](https://novawat.ch/)

It is not permitted to pass on the files, which is why you can only see file changes if you have validated your licence.

The source code is also completely publicly available on GitHub.
